ToolBox/FxcmVolumeDownloader/Program.cs

```diff
@@ -5,12 +5,35 @@
 
 namespace QuantConnect.ToolBox.FxcmVolumeDownload
 {
-    class Program
+    internal class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
+            var isUpdate = false;
+            if (args.Length == 1)
+            {
+                if (args[0] == "all" || args[0] == "update")
+                {
+                    if (args[0] == "update")
+                    {
+                        isUpdate = true;
+                    }
+
+                    args = new[]
+                    {
+                        "EURUSD,USDJPY,GBPUSD,USDCHF,EURCHF,AUDUSD,USDCAD,NZDUSD,EURGBP,EURJPY,GBPJPY,EURAUD,EURCAD,AUDJPY",
```

Comment:
Is this all the symbols that fxcm provides? Just curious why this list is the way it is.

Reply:
Those are the pairs that FXCM provides Volume data. The list was given by mail with the API document.